style.css

```diff
@@ -14,6 +14,239 @@ nav {
   left: 25%;
 }
 
+.works1 {
+  background-color: #dfe1e6;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-auto-rows: minmax(552px, auto);
+  grid-gap: 10px;
+  width: 100%;
+}
+
+.worksec1 {
+  position: static;
+  width: 88%;
+  height: 310px;
+  margin: 25px;
+  left: 50%;
+  top: 40%;
+  border-radius: 9px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  background: white;
+  padding-bottom: 142px;
+  padding-top: 76px;
+}
```

Comment:
- [ ] Great job implementing this class 👍. However, it's one of the requirements, to use `Flexbox` to layout individual cards. Kindly fix this.